heed/src/cookbook.rs

```diff
@@ -0,0 +1,450 @@
+//! A cookbook of examples on how to use heed. Here is the list of the different topics you can learn about:
+//!
+//! - [Listing and Opening the Named Databases](#listing-and-opening-the-named-databases)
+//! - [Create Custom and Prefix Codecs](#create-custom-and-prefix-codecs)
+//! - [Change the Environment Size Dynamically](#change-the-environment-size-dynamically)
+//! - [Decode Values on Demand](#decode-values-on-demand)
+//! - [Advanced Multithreaded Access of Entries](#advanced-multithreaded-access-of-entries)
+//!
+//! # Listing and Opening the Named Databases
+//!
+//! Sometimes it is useful to list the databases available in an environment.
+//! LMDB stores them already in the unnamed database, a database that is always
+//! there in which you can write.
```

Comment:
> LMDB stores them already in the unnamed database, a database that is always there in which you can write.

This sounds strange to my ear 🤔 
Here’s another suggestion

```suggestion
//! Sometimes it is useful to list the databases available in an environment.
//! LMDB automatically stores their names in the unnamed database, a database that doesn't
//! need to be created in which you can write.
```